pkgs/development/libraries/clipper/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl, cmake, ninja, unzip }:
+
+stdenv.mkDerivation rec {
+  version = "6.4.2";
+  name = "Clipper-${version}";
+  src = fetchurl {
+    url = "https://download.sourceforge.net/project/polyclipping/clipper_ver${version}.zip";
```

Comment:
You can use sourceforge `mirror:` protocol.

Reply:
What do you mean exactly? Replace https with mirror?

Reply:
Something like `mirror://sourceforge/polyclipping/clipper_ver${version}.zip`